applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -240,14 +271,14 @@ public static Builder builder() {
      * @throws ServiceRequestException with status code 404 if the {@code key} does not exist.
      */
     public Mono<Response<ConfigurationSetting>> unlockSetting(ConfigurationSetting setting) {
-        ConfigurationSetting result = validateSetting(setting);
+        ConfigurationSetting result = validateSetting(setting, false);
 
         return service.unlockKeyValue(serviceEndpoint, result.key(), result.label(), null, null);
     }
 
     /**
      * Fetches the configuration settings that match the {@code options}. If {@code options} is {@code null}, then all the
-     * {@link ConfigurationSetting}s are fetched in their current state with default fields.
+     * {@link ConfigurationSetting configuration settings} are fetched in their current state.
```

Comment:
"in their current state" is a bit hard to parse for someone unfamiliar with the service. Does it need a clarifying sentence?